tests/h/views/activity_test.py

```diff
@@ -927,6 +927,11 @@ def group(factories):
     factories.Group()
 
     group = factories.Group()
+
+    # Add the group's creator as a member, as would normally be the case for
+    # (non-publisher) groups.
+    group.members.append(group.creator)
+
```

Comment:
Would it make sense to re-structure the group factories? We could get rid of the default `factories.Group` factory and create a `factories.PrivateGroup` which adds the creator to the members inside the factory, so we don't need to care about this in the tests themselves.